api/flaskr/routes/integrations.py

```diff
@@ -22,7 +22,7 @@ def createIntegration():
     content['user'] = user
     integrations_collection.insert(content)
 
-    return Response('Gathered ' + content['name'])
+    return Response('Gathered ' + content['websiteURL'])
```

Comment:
there wasn't a 'name' value on my payload, thus i changed it to websiteURL for the moment.